festim/hydrogen_transport_problem.py

```diff
@@ -208,6 +216,14 @@ def initialise(self):
         self.create_solver()
         self.initialise_exports()
 
+    def create_species_from_trap(self):
```

Comment:
Should we have traps (plural) here instead?

Reply:
sure